docs/update.adoc

```diff
@@ -0,0 +1,35 @@
+= Handling new versions and updates
+:imagesdir: ./assets/images/
+:toc:
+:toc-title:
+:toclevels: 4
+
+== General
+Changing the software version of your app might require a change in agrirouter, if you decide to add new technical message types.
+
+=== When is a new certification version required?
+agrirouter is not interested in any internal changes of your software.
+However, whenever the maximum capabilities of your software change, you'll need to create a new version in the agrirouter UI and run a new certification.
+
+=== What has to be done?
+==== While development
+1. You have to create a new software version in the agrirouter UI
```

Comment:
New "certification version" to use common terms.

Reply:
Agree, done